static/src/javascripts/projects/common/modules/commercial/sticky-ad-banner.js

```diff
@@ -124,11 +124,12 @@ define([
         var transitionTimingFunction = 'cubic-bezier(0, 0, 0, .985)';
         var transitionDuration = '1s';
 
-        els.$header.css({
+        els.$page.css({
+            'will-change': 'transform',
```

Comment:
:bowtie: sweet!

> but at a very high level, will-change is an alias for the old translate3d hack.

When `will-change: transform|opacity`, yes definitely. Not sure it would be smart of the browser to do that when the element's bounding box is part of the document flow though, or would it? For instance, Blink/Webkit will un-promote a node when `will-change: contents`. Basically [all the properties which trigger layout](https://csstriggers.com/).

It really feels like this property should never have been a property in the first place, but rather an API to manipulate the render tree. I'm very curious to see if this will change (no pun intended) with Houdini.